subscriber.rb

```diff
@@ -0,0 +1,52 @@
+require 'aws-sdk-sqs'
+require_relative 'load_aws'
+
+class Subscriber
+
+  attr_reader :sqs
+
+  def initialize
+    creds = Aws::Credentials.new(AWS["access_key_id"], AWS["secret_access_key"])
+    @sqs = Aws::SQS::Client.new(region: AWS["region"], credentials: creds)
+  end
+
+# Todo: URL validation
+  def read_queue(options=[])
+    queue_url = options[0].chomp
+    # Note: AWS SQS wait_time_seconds ranges between 0-20 seconds. 
+    wait_time_seconds = if options[1].nil?
+                          0 
+                        elsif options[1].to_i > 20
+                          puts 'Your wait time must be between 0 to 20 seconds.'
+                          exit(1)
+                        else
+                          options[1].to_i
+                        end
+    
+    loop do
+      result = sqs.receive_message(queue_url: queue_url, max_number_of_messages: 1, wait_time_seconds: wait_time_seconds)
```

Comment:
Lets do this similar to publish messages. Do not use wait_time_seconds, but use it for sleep time.